modules/storage-persistent-storage-reclaim-manual.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+
+[id="reclaim-manual_{context}"]
+= Manually reclaiming a PV
```

Comment:
I see a mixture of the use of "PersistentVolume" and "PV" in module/section headings. For findability, it might be better to title this as "Manually reclaiming a PersistentVolume."